build.gradle

```diff
@@ -119,7 +119,7 @@ def versions = [
         hibernate: '6.0.5.Final',
         jacksonCore: '2.9.8',
         javaxValidation: '2.0.0.Final',
-        puppyCrawl: '7.6',
+        puppyCrawl: '8.17',
```

Comment:
?

Reply:
I reverted this too, works fine with version 7.6